update_script.sh

```
@@ -1,13 +1,2 @@
-sed -i "s|'../../../src/components/chat/ChatRoomEmptyState'|'@/components/chat/ChatRoomEmptyState'|g" frontend/app/\(main\)/chat/\[id\].tsx
-sed -i "s|'../../../src/hooks/useChatRoomSetup'|'@/hooks/useChatRoomSetup'|g" frontend/app/\(main\)/chat/\[id\].tsx
-sed -i "s|'../../../src/utils/chatUtils'|'@/utils/chatUtils'|g" frontend/app/\(main\)/chat/\[id\].tsx
-sed -i "s|'../../../src/components/ChatBubble'|'@/components/ChatBubble'|g" frontend/app/\(main\)/chat/\[id\].tsx
-sed -i "s|'../../../src/components/ChatInputBar'|'@/components/ChatInputBar'|g" frontend/app/\(main\)/chat/\[id\].tsx
-sed -i "s|'../../../src/components/AgentActivityIndicator'|'@/components/AgentActivityIndicator'|g" frontend/app/\(main\)/chat/\[id\].tsx
-sed -i "s|'../../../src/store/useChatStore'|'@/store/useChatStore'|g" frontend/app/\(main\)/chat/\[id\].tsx
-sed -i "s|'../../../src/store/useAgentStore'|'@/store/useAgentStore'|g" frontend/app/\(main\)/chat/\[id\].tsx
-sed -i "s|'../../../src/core/api/ApiService'|'@/core/api/ApiService'|g" frontend/app/\(main\)/chat/\[id\].tsx
-sed -i "s|'../../../src/core/models'|'@/core/models'|g" frontend/app/\(main\)/chat/\[id\].tsx
-sed -i "s|'../../../src/components/agents/QuickViewAgentSheet'|'@/components/agents/QuickViewAgentSheet'|g" frontend/app/\(main\)/chat/\[id\].tsx
+sed -i 's/onPress={onContextPress!}/onPress={onContextPress ?? (() => {})}/g' frontend/src/components/chat/RoomPromptRail.tsx
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)